src/Logging/Connection.php

```diff
@@ -50,30 +50,21 @@ public function exec(string $sql): int
         return parent::exec($sql);
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    public function beginTransaction()
+    public function beginTransaction(): bool
```

Comment:
This change is probably find, but totally unrelated to the issue that you're fixing. Please revert.